docs/spec/auth/05_vesting.md

```diff
@@ -272,7 +272,11 @@ func (cva ContinuousVestingAccount) TrackUndelegation(amount Coins) {
 
 **Note**: If a delegation is slashed, the continuous vesting account will end up
 with an excess `DV` amount, even after all its coins have vested. This is because
-undelegating free coins are prioritized.
+undelegating free coins are prioritized. In addition, the undelegation amount may
+exceed the delegated vesting amount due to the ability of undelegating a fractional
+amount of shares in which case the tokens are truncated and as a result the
```

Comment:
the tokens may be truncated regardless of the ability of undelegating a fractional amount of shares, because a whole amount of shares can still result in a fraction of tokens, and after truncation you end up with a slightly larger ex rate.

Instead, should say, "The undelegation (bond refund) amount may exceed the delegated vesting (bond) amount due to the way undelegation truncates the bond refund, which increases the validator's exchange rate (tokens/shares) slightly."

Also, it should be a separate paragraph from the original Note... Rather than "In addition", I'd just create two notes.

Reply:
Updated.